community/kernel/src/main/java/org/neo4j/kernel/api/Statement.java

```diff
@@ -25,12 +25,10 @@
 /**
  * A statement which is a smaller coherent unit of work inside a {@link KernelTransaction}.
  * There are accessors for different types of operations. The operations are divided into
- * read and write operations, write operations are divided into {@link #dataWriteOperations()} and
- * {@link #schemaWriteOperations()} since there can be only one type of write operations inside
- * any given transaction. In both cases {@link #tokenWriteOperations()} are allowed though,
+ * read and write operations. In both cases {@link #tokenWriteOperations()} are allowed though,
  * which is why it can be accessed separately. The transaction, if still "undecided" about its
- * type of write operations, will be decided when calling either {@link #dataWriteOperations()}
- * or {@link #schemaWriteOperations()}, otherwise if already decided, verified so that it's
+ * type of write operations, will be decided when calling either data operations
+ * or schema operations, otherwise if already decided, verified so that it's
```

Comment:
this not really correct javadoc since there is not schema operations here anymore

Reply:
Hum, I made us a card to update javadocs around `Statement` and `Transaction` when we're done cleaning up. Perfect code freeze activity.